### ISSUE TITLE
Notice: Undefined offset: 2

### ISSUE THREAD

**Issue by sea5kg** [cfortune/PHP-Bounce-Handler#23]
A lot of messages:
Notice: Undefined offset: 2 in PHP-Bounce-Handler/bounce_driver.class.php on line 507

**Comment by mcbmcb0** [comment]
yes me too - this testing on win7-64, xampp and reading email text from a text file
however a bit more luck with this fork: [(https://github.com/rbairwell/PHP-Bounce-Handler)]
thanks to Chris his work on this

**Comment by mishazharov** [comment]
Same here, 64 bit linux, Php v. 5.3.29. Going to check the other fork out, thanks
